src/App.tsx

```diff
@@ -15,9 +18,29 @@ function useForceUpdate() {
 }
 
 const App: React.FC = () => {
+  const bapsConnectionState = useSelector(
+    (state: RootState) => state.bapsSession.connectionState
+  );
+
   const [inputVal, setInputVal] = useState("");
   const force = useForceUpdate();
 
+  const {
+    currentUser,
+    userLoading,
+    currentTimeslot,
+    timeslotLoading,
+  } = useSelector((state: RootState) => state.session);
+
+  if (process.env.REACT_APP_BAPSICLE_INTERFACE) {
+    document.title = "BAPS3 Presenter";
+    if (bapsConnectionState !== "CONNECTED") {
+      return <BAPSSessionHandler />;
+    } else {
+      return <Showplanner timeslotId={-1} />;
```

Comment:
What's the significance of -1 here?

Reply:
That it required a number type for WebStudio use :P Probably can be made optional.